frontend/src/pedigree.tsx

```diff
@@ -243,3 +259,121 @@ export function herdPedigree(genebanks: Genebank[], herdId: string | undefined,
 
   return {nodes: nodes, edges: edges}
 }
+
+/**
+ * Return the edges that connect ancestor id to seeked descendant id
+ * @param edges the edges that make up the connections between the nodes in the pedigree
+ * @param ancestorId the ancestor id
+ * @param seekedDescendantId the child id
+ */
+export function getConnectingEdges(edges: Edge[], ancestorId:string, seekedDescendantId:string): Set<string> {
+  const getChildren = (edges: Edge[], parentId: string, seekeDescendantId: string) => {
+    let connectingEdges = new Set<string>()
+    let seekedFound: boolean = false
+    edges.forEach(edge => {
+      // Current node is a child to parentNode
+      if (edge.to == parentId) {
+        // Current node is the seeked node, add to connectingEdges
+        if (edge.from == seekeDescendantId) {
+          connectingEdges.add(edge.id)
+          seekedFound = true
+          return
+        } else {
+          let res = getChildren(edges, edge.from, seekeDescendantId)
+          // Descendant node connected to seeked node, add current node
+          if (res.seekedFound) {
+            connectingEdges.add(edge.id)
+            connectingEdges = new Set([...connectingEdges, ...res.connectingEdges])
+            seekedFound = true
+            return
+          }
+      }
+        }
+      }
+    )
+    return {connectingEdges: connectingEdges, seekedFound: seekedFound}
+  }
+
+  const finalRes = getChildren(edges, ancestorId, seekedDescendantId)
+  let finalConnectingEdges = finalRes.connectingEdges
+  return finalConnectingEdges
+}
+
+/**
+ * Returns the combined pedigree of `parents` from the data in `genebanks`
+ * and a potential offspring as a child node of the parents
+ * @param genebanks the genebank data to use for pedigree generation
+ * @param parents the parents whos pdeigree should be plotted
+ * @param generations the number of generations to plot. The potential 
+ * offspring is not taken into account as a generation, i.e. it is the 
+ * number of generations from the parents 
+ */
+export function parentPedigree(genebanks: Genebank[], parents: Individual[], generations: number = 1000): Pedigree {
+  let nodes: Node[] = []
+  let edges: Edge[] = []
+
+  const offspringNode = {id: "13371337",
+    x: 0,
+    label: "Potentiell avkomma",
+    shape: 'triangle',
+    color:  {
+      border: 'darkgrey',
+      background: '#e5dbd7',
+      highlight: {
+        border: 'grey',
+        background: '#b5aeab'
+      },
+      hover: {
+        border: 'black',
+        background: '#b5aeab'
+      }
+    }
+  }
+  nodes.push(offspringNode)
+
+  parents.forEach((parent: Individual) => {
+    edges.push({id: `${offspringNode.id}-${parent.number}`,
+                                  from: offspringNode.id,
+                                  to: parent.number})
+    //FEEDBACK, should it be optional to use calcPedigreeDan as well?
```

Comment:
we should put in the time to make a better calcPedigree function - none of the current ones should matter (imho). :)